WinUIGallery/Samples/Data/ControlInfoData.json

```diff
@@ -3692,6 +3692,70 @@
           ]
         }
       ]
+    },
+    {
+      "UniqueId": "Notifications",
+      "Title": "Notifications",
+      "ImagePath": "",
+      "IconGlyph": "",
+      "Items": [
+        {
+          "UniqueId": "AppNotification",
```

Comment:
For the sake of ensuring that the NavView itemslist does not become toooo big: could we move these under `System`?